src/back/auth/src/api/logout_callback.cpp

```diff
@@ -11,13 +11,17 @@ LogoutCallback::LogoutCallback(
 	const components::ComponentContext& ctx)
 	: server::handlers::HttpHandlerBase{conf, ctx}
 	, _s{ctx.FindComponent<Service>()}
+	, _mapHttpMethodToSchema{LoadSchemas(kName, _s.GetJSONSchemasPath())}
 {}
 
 std::string LogoutCallback::HandleRequestThrow(
 	const server::http::HttpRequest& req,
 	server::request::RequestContext&) const
 {
-	auto url = getCallerUrl(req);
+	const formats::json::Value body;
+	const auto params = ValidateRequest(_mapHttpMethodToSchema, req, body);
```

Comment:
Там есть перегруженная функция
```suggestion
	const auto params = ValidateRequest(_mapHttpMethodToSchema, req);
```

Reply:
Готово (везде).